website/util/__init__.py

```diff
@@ -69,7 +70,7 @@ def api_url_for(view_name, _absolute=False, _xml=False, *args, **kwargs):
 def api_v2_url(path_str,
                params=None,
                base_route=website_settings.API_DOMAIN,
-               base_prefix='',
+               base_prefix=api_settings.API_BASE,
```

Comment:
That's an interesting alternate proposal. But it's worth noting that the keyword argument was originally designed specifically for use in tests (to keep them config-file-independent): 
https://github.com/CenterForOpenScience/osf.io/blob/develop/tests/test_utils.py#L66

What are the advantages of essentially monkey-patching the variable in this context?